Dependencies/UrlLib/Source/Windows/UrlRequest.cpp

```diff
@@ -114,6 +115,11 @@ namespace UrlLib
 
                             m_responseUrl = winrt::to_string(responseMessage.RequestMessage().RequestUri().RawUri());
 
+                            for (auto&& iter : responseMessage.Headers())
+                            {
+                                m_headers.insert(std::make_pair(winrt::to_string(iter.Key()), winrt::to_string(iter.Value())));
+                            }
```

Comment:
Would it be better to store the `Web::Http::Headers::HttpResponseHeaderCollection` directly to avoid copying the strings if no one ever needs them?

Reply:
The header collection becomes invalid outside of the task. What's the good practice to keep a reference on it? If it needs a copy, then doing it with a std::map is more consistent with the rest of the class IMO.

Reply:
> The header collection becomes invalid outside of the task

It's not ref counted like normal WinRT object? This isn't that big a deal either way.